scripts/flytekit_spark_entrypoint.sh

```diff
@@ -88,30 +88,30 @@ case "$SPARK_K8S_CMD" in
       "$SPARK_HOME/bin/spark-submit"
       --conf "spark.driver.bindAddress=$SPARK_DRIVER_BIND_ADDRESS"
       --deploy-mode client
-      "$@" "$PYSPARK_PRIMARY" "$PYSPARK_ARGS"
+      "$@" $PYSPARK_PRIMARY $PYSPARK_ARGS
     )
     ;;
     driver-r)
     CMD=(
       "$SPARK_HOME/bin/spark-submit"
       --conf "spark.driver.bindAddress=$SPARK_DRIVER_BIND_ADDRESS"
       --deploy-mode client
```

Comment:
Lets put comments over here explaining what we are doing. Else this is hard to follow